src/index.ts

```diff
@@ -246,9 +252,19 @@ export class DebugProxy extends EventEmitter implements DebugProxyInterface {
     const possiblyHitBreakpointList = await Promise.all(possiblyHitPromiseList);
     possiblyHitBreakpointList.forEach((breakpoint: Breakpoint) => {
       if (breakpoint.isFinalState) {
-        assert(this.breakpointInfoMap.has(breakpoint.id));
-        this.breakpointInfoMap.get(breakpoint.id)![hit] = true;
-        hitAny = true;
+        const breakpointInfo = this.breakpointInfoMap.get(breakpoint.id);
+        if (!breakpointInfo) {
+          throw new Error(
+              `The breakpoint with ID ${breakpoint.id} ` +
```

Comment:
You are correct, and this is a general constraint. I took inspiration from the web UI, which only shows snapshots set in the web UI by default.

I think this reduces the risk of deleting someone else's breakpoint and the confusion of having random new breakpoints show up, if multiple developers are debugging the same app. However, an option to have all breakpoints would be useful, so I've raised #19.

Reply:
In general I think it's a good default but we need to allow developers to share breakpoints easily.  As for deleting each others breakpoints it's a general risk with any resource that is exposed.  In this case IAM is the control for what each developer can do.  I think it would be good to let that happen if people have access to add and remove breakpoints we probably shouldn't be a control point.

Reply:
Sounds good. I will focus on the DevTools integration for the rest of this week, and addressing #19 will be a priority after that.